R/pkg/R/sparkR.R

```diff
@@ -564,6 +564,23 @@ setJobDescription <- function(value) {
   invisible(callJMethod(sc, "setJobDescription", value))
 }
 
+#' Set a local property that affects jobs submitted from this thread, such as the
+#' Spark fair scheduler pool.
```

Comment:
I tried to make a better description for a while but then just ended up with copying Python's: https://github.com/apache/spark/blob/209b9361ac8a4410ff797cff1115e1888e2f7e66/python/pyspark/context.py#L934-L939